azure-pipelines-gitTests-template.yml

```diff
@@ -10,7 +10,7 @@ parameters:
   - name: REPO_COUNT
     displayName: Repo Count
     type: number
-    default: 200
+    default: 300
```

Comment:
This value is basically always overridden by other workflows, and the triggerer sets this; this probably won't have an effect, no?

Reply:
> This value is basically always overridden by other workflows, and the triggerer sets this

I think that the tasks with a `schedule` that use this template just don't set the `REPO_COUNT`:

https://github.com/microsoft/typescript-error-deltas/blob/21609ac5683ae788aa56bde7e740dfab7bdc7622/azure-pipelines-gitTests-tsserver-js.yml#L17-L20

and

https://github.com/microsoft/typescript-error-deltas/blob/main/azure-pipelines-gitTests-tsserver-ts.yml#L17-L20

Reply:
oh, gotcha